charts/otomi-pipelines/templates/tekton-otomi-task.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: otomi-task
+  namespace: otomi-pipelines
+spec:
+  params:
+    - name: OTOMI_VERSION
+    - name: CI
+    - name: repoUrl
+    - name: giteaCredentialsSecretName
+  workspaces:
+    - name: source
+      mountPath: /home/app/stack/env
+  stepTemplate:
+    resources: {{- toYaml .Values.tektonTask.resources | nindent 6 }}    
+    env:
+    - name: GITEA_USERNAME
+      valueFrom:
+        secretKeyRef:
+          name: '$(params.giteaCredentialsSecretName)'
+          key: username
+    - name: GITEA_PASSWORD
+      valueFrom:
+        secretKeyRef:
+          name: '$(params.giteaCredentialsSecretName)'
+          key: password
+    - name: AZURE_CLIENT_ID
```

Comment:
The `otomi-sops-secrets` can contain different env depending on the KMS provider.
It would be the best if it is possible to export the content of `otomi-sops-secrets`  without naming particular envs

Reply:
solved by using envFrom to load sops secret.